src/iris/api.py

```diff
@@ -684,7 +685,14 @@ def debug_auth(self, req, resp, resource, params):
 
         # Proceed with authenticating this route as a third party application
         try:
-            app, client_digest = req.get_header('AUTHORIZATION', '')[5:].split(':', 1)
+            # Ignore HMAC requirements for custom webhooks
+            if req.env['PATH_INFO'].startswith('/v0/webhooks/'):
+                app = req.get_param('application', required=True)
+                if not app:
```

Comment:
When `required=True`, Falcon will raise a 4xx if the param is not provided. You should be able to remove the `if not app` check